deepmd/tf/utils/stat.py

```
@@ -0,0 +1,214 @@
+    compute_stats_from_redu,
+)
+from deepmd.utils.path import (
+    DPPath,
+)
+
+log = logging.getLogger(__name__)
+
+
+def _restore_from_file(
+    stat_file_path: DPPath | None,
+    keys: list[str] = ["energy"],
```

Ruff (0.15.17)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)